rubocop.yml

```diff
@@ -90,6 +95,8 @@ Style/MutableConstant:
   Enabled: false
 Style/NumericPredicate:
   Enabled: false
+Style/OptionHash:
```

Comment:
```ruby
class Bla
  def test(options = {})
    other({ a: 1 }.merge(options))
  end

  def other(a:, b: 2, c: 3)
    puts "#{a} #{b} #{c}"
  end
end
```

this is one example where merging is convenient and you wouldn't want to duplicate the method definition everywhere.

another case is when you have a large hash which, converted to keyword arguments, would make rubocop complain again.